docs/user_guide/assessment.rst

```diff
@@ -42,17 +156,30 @@ of the ten cases where the true value is `1`, so we expect the recall to be 0.5:
 
 .. _metrics_with_grouping:
 
-Metrics with Grouping
-^^^^^^^^^^^^^^^^^^^^^
+Metrics with grouping using :code:`MetricFrame`
+-----------------------------------------------
 
-When considering fairness, each row of input data will have an associated
+In a typical fairness assessment, each row of input data will have an associated
 group label :math:`g \in G`, and we will want to know how the metric behaves
-for each :math:`g`. To help with this, Fairlearn provides a class, which takes
-an existing (ungrouped) metric function, and applies it to each group within a
+for each :math:`g`. To help with this, Fairlearn provides a class that takes
+an existing (ungrouped) metric function and applies it to each group within a
```

Comment:
Perhaps mention "like scikit-learn's metrics"?

Reply:
Added a reference to :module:`sklearn.metrics`, but let me know if that's not the correct domain directive.

Reply:
@hildeweerts I have addressed both your comments! I added the function references and replaced "ungrouped" with "disaggregated" to more directly link that sentence to the section above.